sonar-plugin-api/src/main/java/org/sonar/api/profiles/RulesProfile.java

```diff
@@ -217,22 +216,25 @@ public RulesProfile setLanguage(String s) {
   }
 
   /**
-   * For internal use only.
+   * Does nothing.
    *
-   * @since 2.5
+   * @return {@code null}
+   * @deprecated in 6.5
    */
+  @Deprecated
   @CheckForNull
   public String getParentName() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)